app/R/hard_load.R

```diff
@@ -0,0 +1,41 @@
+get_parameters = function() {
+  params = list(modelyear = 2024, ACC_LevyRate = 0.0153, ACC_MaxLeviableIncome = 139384, 
+                Accommodation_AbatementRate = 0.25, Accommodation_BaseRateThreshold_Mortgage = 0.3, 
+                Accommodation_BaseRateThreshold_Rent = 0.25, Accommodation_MaxRate_CoupleDeps_Single2_Deps_Area1 = 0, 
```

Comment:
Most of the Accommodation Supplement parameters are incorrectly 0, which I think is an error you might have inherited from somewhere else.